spec/support/hanami_commands.rb

```diff
@@ -15,6 +15,7 @@ def rackup(args = {}, &blk)
       end
 
       def server(args = {}, &blk)
+        args[:port] ||= rand(1000..9000)
```

Comment:
Good point, though I think primes might be overkill. Maybe we can skip anywhere the last digit is 0, or is a one or two digit number repeated? e.g. `4242`?